packages/core/src/external/cda/remove-b64.ts

```diff
@@ -37,18 +47,39 @@ export function removeBase64PdfEntries(payloadRaw: string): {
         if (comp.section.entry) {
           //eslint-disable-next-line @typescript-eslint/no-explicit-any
           comp.section.entry = toArray(comp.section.entry).filter((entry: any) => {
-            const mediaType = entry.act?.text?.["_mediaType"]?.trim().toLowerCase();
-            if (
-              (BINARY_MIME_TYPES.includes(mediaType) || mediaType == undefined) &&
-              (entry.act?.text?.["_representation"]?.trim().toLowerCase() ===
-                b64Representation.toLowerCase() ||
-                entry.organizer?.component?.observationMedia?.value?.["_representation"]
-                  ?.trim()
-                  .toLowerCase() === b64Representation.toLowerCase())
-            ) {
-              isRemovedEntries = true;
-              b64Attachments.push(entry);
-              return false;
+            if (isConcernActEntry(entry)) {
+              const mediaType = entry.act?.text?.["_mediaType"]?.trim().toLowerCase();
```

Comment:
shouldn't act be defined if `isConcernActEntry` is true? Maybe TS isn't smart enough to realize though...

Reply:
yes it is smart enough! im actually a bit surprised haha